core/src/test/java/io/grpc/internal/ManagedChannelImplTest.java

```diff
@@ -4053,15 +4059,6 @@ public String getDefaultScheme() {
     assertThat(args).isNotNull();
     assertThat(args.getDefaultPort()).isEqualTo(DEFAULT_PORT);
     assertThat(args.getProxyDetector()).isSameInstanceAs(neverProxy);
-
-    verify(offloadExecutor, never()).execute(any(Runnable.class));
```

Comment:
This is the black-box form of the verification. And it is testing another API. We shouldn't delete this.